projects/fal/src/fal/cli/apps.py

```diff
@@ -62,9 +64,13 @@ def _list(args):
         else:
             apps.sort(key=lambda x: x.alias)
 
-        table = _apps_table(apps)
-
-    args.console.print(table)
+        if args.output == "pretty":
+            table = _apps_table(apps)
+            args.console.print(table)
+        elif args.output == "json":
+            apps_as_dicts = [asdict(a) for a in apps]
+            res = json.dumps(apps_as_dicts)
+            args.console.print(res)
```

Comment:
this prints a list, let's print a dict with `"apps": <the list>` instead in case we want to add more info the future to not break tools that expect a list.

`{"apps" [], "stats": ...}` can be added later and not break anything (stats is an example)